packages/dev/sharedUiComponents/src/stringTools.ts

```diff
@@ -46,7 +46,7 @@ export class StringTools {
                 if (popup) {
                     popup.location.href = url;
                 } else {
-                    location = url;
+                    location.href = url;

```

Comment:
in this case should we not update 47 as well ?

Reply:
Nothing is wrong in 47. Or am i missing something?

Reply:
ahah I was reading a removed href instead of added.... stupid me...